### ISSUE TITLE
Calibration Equipment

### ISSUE THREAD

**Issue by chad-m-carlson** [chad-m-carlson/arrow-tech#42]
A few things I will need to know before I get started setting it up so you can change the calibration equipment certifications yourself.

1.  Do the J.L. Shepherd 20 and TEMCO 100 always have the same tracking number and date?
2.  Will these two units always be the same or subject to change?
3.  Is there ever a need to change the exposure rates?  
      3a. Would it be helpful to have only the exposure rates the TEMCO 100 is capable of in the drop down if that is what is selected?

**Comment by btc721** [comment]
1. No. They will change every 6 months or so. 
2. The units will always be the same.
3. The exposure rates will change whenever the source is calibrated.  Every 2-4 months or so. 
3a. Yes.

**Comment by chad-m-carlson** [comment]
What I meant was will the two pieces of equipment always have the same calibration cert? I know it will change, but will they both be on the same one?

**Comment by btc721** [comment]
They will always be on the same one.

**Comment by chad-m-carlson** [comment]
So the exposure rates change separately than the calibration equipment calibrations?

**Comment by btc721** [comment]
Yes, the exposure rates only change when we actually calibrate our source.  The date's are for when our standard dosimeters are calibrated by Batelle.   Our standards are what we use to calibrate the source.
